### PR TITLE
Ptdev

### PR DESCRIPTION
**Pull request by lifeix**
two minor refine about ptdev

1. rename ptirq_lookup_entry_by_sid to ptirq_lookup_entry
2. minor refine about ptirq_build_physical_msi

Tracked-On: #4550
Signed-off-by: Li Fei1 <fei1.li@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
2 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:common filename:/hypervisor/arch/x86/guest/assign.c func:ptirq_lookup_entry offset:83<br> reason:Pointer assignment to wider scope.<br><2> config:common filename:/hypervisor/arch/x86/guest/assign.c func:ptirq_softirq offset:562<br> reason:Implicit conversion: actual to formal param (MR). :  (int and unsigned long):  param  3<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: hypervisor/arch/x86/guest/assign.c [hunk 1]
```diff
@@ -59,7 +59,7 @@ static struct acrn_vcpu *is_single_destination(struct acrn_vm *vm, const struct
  * vm must not be NULL when lookup by virtual sid.
  */
 static inline struct ptirq_remapping_info *
-ptirq_lookup_entry_by_sid(uint32_t intr_type,
+ptirq_lookup_entry(uint32_t intr_type,
```
Comment (dongyaozu) [inline hypervisor/arch/x86/guest/assign.c:62]: find_ptirq_entry?

Comment (lifeix) [reply hypervisor/arch/x86/guest/assign.c]: OK

**Comment by HuiHuangShi** [comment]
2 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:common filename:/hypervisor/arch/x86/guest/assign.c func:find_ptirq_entry offset:83<br> reason:Pointer assignment to wider scope.<br><2> config:common filename:/hypervisor/arch/x86/guest/assign.c func:ptirq_softirq offset:562<br> reason:Implicit conversion: actual to formal param (MR). :  (int and unsigned long):  param  3<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
2 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:common filename:/hypervisor/arch/x86/guest/assign.c func:find_ptirq_entry offset:83<br> reason:Pointer assignment to wider scope.<br><2> config:common filename:/hypervisor/arch/x86/guest/assign.c func:ptirq_softirq offset:562<br> reason:Implicit conversion: actual to formal param (MR). :  (int and unsigned long):  param  3<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**